README.md

```diff
@@ -137,6 +137,8 @@ In order to extend our usage of `webpack`, you can define a function that extend
 // This file is not going through babel transformation.
 // So, we write it in vanilla JS
 // (But you could use ES2015 features supported by your Node.js version)
+const webpack = require('webpack')
```

Comment:
Webpack is actually not required unless you need to configure some webpack specific plugins.
Nevertheless I merged this PR since I believe it doesn't harm and some users where confused by seeing a webpack config, without webpack. 
So feel free to omit it in your case.